controllers/storagecluster/exporter.go

```diff
@@ -23,15 +24,22 @@ const (
 	scrapeInterval       = "1m"
 )
 
-var exporterLabels = map[string]string{
+var exporterServiceMonitorLabels = map[string]string{
+	"app.kubernetes.io/component": exporterName,
+	"app.kubernetes.io/name":      exporterName,
+}
+
+// do not use the exporterServiceMonitorLabels maps as it will be expanded while reconcile.
+// We should have made a constant for exporterServiceLabels, but we can't do that in golang.
+var exporterServiceLabels = map[string]string{
```

Comment:
Let's keep this as 1 variable. While creating ServiceMonitor we can merge this with other labels and use it.
This duplication isn't needed.

Reply:
done